herd/tests/instructions/AArch64.self/A016.litmus

```diff
@@ -0,0 +1,13 @@
+AArch64 A016
```

Comment:
Yes and no. No in the sense that I did not actually manage to produce a different output with herd7. Yes in the sense that the actual states herd7 goes through are different. The objective behind A016 was to capture how the label-processing logic in `pseudo.ml` might result in different outcomes when computing what used to be "code_instr" in `AArch64Sem.ml`. In particular, the first fix I rolled out was flawed from that perspective, and then I silently force-pushed a better one.

However, this does not actually matter at all, if indeed the "code_instr" set computed in `get_overwriting_instructions` of `AArch64Sem.ml` was simply redundant.

I thought the test could be added anyway just in case, since it is a result of a (failed) intent. I'd be happy to remove it, if it seems unnecessary.

Reply:
Thanks!